hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/TargetRepository.java

```diff
@@ -59,6 +59,29 @@ void setAssignedDistributionSetAndUpdateStatus(@Param("status") TargetUpdateStat
             @Param("lastModifiedBy") String modifiedBy, @Param("targets") Collection<Long> targets);
 
     /**
+     * Sets {@link JpaTarget#getAssignedDistributionSet()} and
```

Comment:
Better:

Sets {@link JpaTarget#getAssignedDistributionSet()}, {@link JpaTarget#getInstalledDistributionSet()} and
{@link JpaTarget#getInstallationDate()}.